generator/src/main/java/com/graphicsfuzz/generator/tool/Generate.java

```diff
@@ -157,6 +160,13 @@ public static void addGeneratorCommonArguments(ArgumentParser parser) {
     parser.addArgument("--shader-stage")
         .help("Only fuzz the given shader stage.")
         .type(ShaderKind.class);
+
+    parser.addArgument("--push-constant-probability")
+        .help("Probability of converting a random uniform to push constant; "
```

Comment:
Maybe add that this is ignored if the shading language version does not support push constants.

Reply:
Added.